server/server/src/router_service.rs

```diff
@@ -268,6 +269,47 @@ pub fn google_play_notification_webhooks(
         })
 }
 
+static APP_STORE_WEBHOOK_ROUTE: &str = "app_store_notification_webhook";
+pub fn app_store_notification_webhooks(
+    server_state: &Arc<ServerState>,
+) -> impl Filter<Extract = impl warp::Reply, Error = warp::Rejection> + Clone {
+    let cloned_state = server_state.clone();
+
+    warp::post()
+        .and(warp::path(APP_STORE_WEBHOOK_ROUTE))
+        .and(warp::any().map(move || cloned_state.clone()))
+        .and(warp::body::bytes())
+        .then(|state: Arc<ServerState>, body: Bytes| async move {
+            let span = span!(
```

Comment:
Can you make sure all the webhooks are getting timed? Just once over and compare to the main `core_req` and make sure we're doing everything the same way, logging & timing. Make sure we're creating spans the same way and that we're timing the same way.